lms/security.py

```diff
@@ -110,19 +142,25 @@ def identity(self, request):
 
         return Identity("", [])
 
-    def permits(self, request, context, permission):
-        return _permits(self, request, context, permission)
+    def permits(self, request, _context, permission):
+        return _permits(self.identity(request), permission)
 
 
-def _permits(policy, request, _context, permission):
-    identity = policy.identity(request)
+def _permits(identity, permission):
```

Comment:
`permits` only takes the arguments it needs

Reply:
:+1: